apps/www/components/footer/footer.tsx

```diff
@@ -13,6 +13,7 @@ const navigation = [
   {
     title: "Company",
     links: [
+      { title: "Roadmap", href: "/roadmap" },
```

Comment:
Let's move this link to after "Templates" 

About, Blog, Changelog, and Templates are definitely more important public company information than the roadmap